### DISCUSSION TITLE
Feature request: add "date" column option in queue table menu

### DISCUSSION THREAD

**Discussion by rare-magma** [jcorporation/myMPD#818] [Q&A] [answered]
Thanks for this great piece of software!

I'd like to know if it would be possible to add the option to display the track's date in the queue view via an option in the display columns menu?

Cheers!

**Comment by jcorporation** [comment, marked as answer]
If you enable the tags in MPD and in myMPD they should show up in the columns menu.

**Comment by rare-magma** [comment]
oh I wasn't aware of that feature, thanks @jcorporation !
I've added the date tag and I can see it now in the queue columns after adding it in the menu.
Unfortunately it seems that the request for the queue doesn't include the new tag so the value is always "-":
```
{"jsonrpc":"2.0","id":0,"method":"MYMPD_API_QUEUE_LIST","params":{"offset":0,"limit":25,"cols":["Title","Artist","Album","Duration","AlbumArtist","Genre"]}}
```

**Reply by jcorporation** [reply]
This is a bug: #822
